Assets/Scripts/CardController.cs

```diff
@@ -4,15 +4,26 @@
 
 public class CardController : MonoBehaviour
 {
+
+    public HandController hand;
+
+    SpriteRenderer spriteRenderer;
+
+
```

Comment:
Nit: Is this extra space necessary?

Reply:
No, and actually I want to keep private and public variables separate. Corrected.